docs/src/modules/operations/pages/regions/index.adoc

```diff
@@ -42,12 +42,12 @@ Stateful components like Event Sourced Entities and Workflows can be replicated
 
 There are two operational choices for deciding where the primary is located:
 
-* *Static mode* - one region is defined as the primary for the project, and all stateful component instances will use that region as primary
-* *Dynamic mode* - the primary is selected by each individual component instance based on where the first event is persisted
+* *pinned-region* mode - one region is defined as the primary for the project, and all stateful component instances will use that region as primary
+* *request-region* mode - the primary is selected by each individual component instance based on where the write requests occur
```

Comment:
Maybe try the definition styling here (with `::`)

Reply:
this is just a short description and I think bullets work fine here